modules/cnf-performing-end-to-end-tests-running-the-tests.adoc

```diff
@@ -29,31 +24,18 @@ You provide the test image with a `kubeconfig` file in current directory and its
 [source,terminal,subs="attributes+"]
 ----
 $ podman run -v $(pwd)/:/kubeconfig:Z -e KUBECONFIG=/kubeconfig/kubeconfig \
--e LATENCY_TEST_RUN=true -e DISCOVERY_MODE=true -e FEATURES=performance registry.redhat.io/openshift4/cnf-tests-rhel8:v{product-version} \
-/usr/bin/test-run.sh -ginkgo.focus="\[performance\]\ Latency\ Test"
+-e LATENCY_TEST_RUNTIME=<time in seconds>\
+-e MAXIMUM_LATENCY=<time in microseconds> \
+registry.redhat.io/openshift4/cnf-tests-rhel8:v{product-version} /usr/bin/test-run.sh \
+--ginkgo.v
 ----
 
-. Optional: Append `-ginkgo.dryRun` to run the latency tests in dry-run mode. This is useful for checking what the tests run.
-
-. Optional: Append `-ginkgo.v` to run the tests with increased verbosity.
+. Optional: Append `--ginkgo.dryRun` to run the latency tests in dry-run mode. This is useful for checking what the tests run.
```

Comment:
Maybe, if possible, add a clarification after "what", such as "what commands the tests run"? Not critical

Reply:
makes sense